core/src/main/java/org/pentaho/platform/scheduler2/quartz/EmbeddedQuartzSystemListener.java

```diff
@@ -163,6 +163,11 @@ public boolean startup( final IPentahoSession session ) {
     return result;
   }
 
+  protected void initializeScheduler()
+    throws org.pentaho.platform.api.scheduler2.SchedulerException {
+    PentahoSystem.get( IScheduler.class ).start();
+  }
```

Comment:
Also the name `initializeScheduler()` is confusing and/or misleading as line previous to it https://github.com/pentaho/pentaho-scheduler-plugin/blob/scheduler-plugin/core/src/main/java/org/pentaho/platform/scheduler2/quartz/EmbeddedQuartzSystemListener.java#L112 already gets an instance of `IScheudler.class` and calls method on the returned class